winui/AvatarView/getting-started.md

```diff
@@ -1,10 +1,77 @@
 ---
 layout: post
-title: Getting Started with WinUIAvatar View control | Syncfusion
-description: Learn all about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control, its elements, and more here.
+title: Getting Started with WinUI AvatarView control | Syncfusion
+description: Learn here all about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control with its basic features.
```

Comment:
Description: Learn about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control with its basic features.